test/e2e-test.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+#######################################################################
+assert ()                 #  If condition false,
+{                         #+ exit from script
+                          #+ with appropriate error message.
+  E_PARAM_ERR=98
+  E_ASSERT_FAILED=99
+
+
+  if [ -z "$2" ]          #  Not enough parameters passed
+  then                    #+ to assert() function.
+    return $E_PARAM_ERR   #  No damage done.
+  fi
+
+  lineno=$2
+
+  if [ ! $1 ] 
+  then
+    echo "Assertion failed:  \"$1\""
+    echo "File \"$0\", line $lineno"    # Give name of file and line number.
+    exit $E_ASSERT_FAILED
+  # else
+  #   return
+  #   and continue executing the script.
+  fi  
+} # Insert a similar assert() function into a script you need to debug.    
+#######################################################################
+
+
+export MAKEFIRSTCONTRIBUTION=yes
+
+sed -i 's/const REQUIRED_POWER: usize = [0-9][0-9];*/const REQUIRED_POWER: usize = 8;/g' /app/src/bn256/mod.rs
```

Comment:
maybe you can put each test case into a function that describes what you are testing.

Reply:
I do not have different test cases. Alex is switching in his code compute_constrainted the different libraries, like this:
```
// use powersoftau::bn256::{Bn256CeremonyParameters};
use powersoftau::small_bn256::{Bn256CeremonyParameters};
```
Hence I thought it is good to set both values.